src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue28530.cs

```diff
@@ -0,0 +1,23 @@
+#if TEST_FAILS_ON_WINDOWS
+// https://github.com/dotnet/maui/issues/13027 In windows, .NET MAUI CollectionView does not reorder when grouped
+using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues;
+public class Issue28530 : _IssuesUITest
+{
+	public Issue28530(TestDevice device) : base(device) { }
+
+	public override string Issue => "[Catalyst] CanMixGroups Set to False Still Allows Reordering Between Groups in CollectionView";
+
+	[Test]
+	[Category(UITestCategories.CollectionView)]
+	public void ReorderBetweenGroupsShouldNotOccurWhenCanMixGroupsIsFalse()
+	{
+		App.WaitForElement("CollectionViewControl");
+		App.DragAndDrop("Item 2", "Item 3");
+		VerifyScreenshot();
```

Comment:
Fails on iOS:
`Snapshot different than baseline: ReorderBetweenGroupsShouldNotOccurWhenCanMixGroupsIsFalse.png (1.44% difference)`
![image](https://github.com/user-attachments/assets/3a621926-c28e-4e2e-a3ab-994dbc248425)

Reply:
I have resaved the iOS image from CI. Could you please verify it now?  @jsuarezruiz